libs/datasets/sources/can_scraper_state_providers.py

```diff
@@ -1,11 +1,13 @@
 from covidactnow.datapublic.common_fields import CommonFields
+from scripts import update_can_scraper_state_providers
```

Comment:
I'm a bit hesitant to import the `scripts` package directly.  I don't think it's clear that it comes from `covid-data-public` and probably more important is that I don't think it's obvious that if you change code in covid-data-public/scripts you'll potentially be breaking upstream code in covid-data-model.  

I'd be happy to meet in the middle somewhere - maybe we can provide some base class that provides a well defined transform interface that we lean on a bit more?  Thoughts on that concern?

Reply:
Also I think the only reason that we can even see the scripts directory is an artifact of the way we install the package.  If it were a proper pip package (or we installed it another way than referring to the path of covid-data-public directly) i don't think it would work

Reply:
We want to move ccd_helpers from covid-data-public to covid-data-model. I don't see much value in refactoring to support sharing this code between the repos because I don't see anything blocking the move. I see merge this today and move ccd_helpers tomorrow.